### PR TITLE
Updated for Beta15

### PR DESCRIPTION
**Pull request by paynerc**

### CODE REVIEW

File: ObjCVideoQuickstart/ViewController.m [hunk 1]
```diff
@@ -142,10 +142,10 @@ - (void)startPreview {
...
 - (void)flipCamera {
-    if (self.camera.source == TVIVideoCaptureSourceFrontCamera) {
-        [self.camera selectSource:TVIVideoCaptureSourceBackCameraWide];
+    if (self.camera.source == TVICameraCaptureSourceFrontCamera) {
+        [self.camera selectSource:TVICameraCaptureSourceBackCameraWide];
     } else {
-        [self.camera selectSource:TVIVideoCaptureSourceFrontCamera];
+        [self.camera selectSource:TVICameraCaptureSourceFrontCamera];
     }
 }
```
Comment (ceaglest) [inline ObjCVideoQuickstart/ViewController.m]: Adjust mirroring of TVIVideoView here, or in the TVICameraCapturerDelegate callback?

**Comment by paynerc** [comment]
Now responding to the `cameraCapturer:didStartWithSource:` delegate call and setting mirroring as appropriate.

File: ObjCVideoQuickstart/ViewController.m [hunk 2]
```diff
@@ -142,24 +138,28 @@ - (void)startPreview {
...
 
 - (void)prepareLocalMedia {
```
Comment (ptankTwilio) [inline ObjCVideoQuickstart/ViewController.m]: s/prepareLocalMedia/prepareLocalTracks

File: ObjCVideoQuickstart/ViewController.m [hunk 3]
```diff
@@ -142,24 +138,28 @@ - (void)startPreview {
...
     
-    // Adding local audio track to localMedia
+    // Create an audio track.
     if (!self.localAudioTrack) {
-        self.localAudioTrack = [self.localMedia addAudioTrack:YES];
+        self.localAudioTrack = [TVILocalAudioTrack track];
+
+        if (!self.localAudioTrack) {
+            [self logMessage:@"Failed to add audio track"];
+        }
```
Comment (ptankTwilio) [inline ObjCVideoQuickstart/ViewController.m]: localAudioTrack is of no use when client is not connected to a Room. Should we demonstrate deleting audioTrack in didDisconnect callback?

File: ObjCVideoQuickstart/ViewController.m (hunk 2 above)
Comment (ceaglest) [inline ObjCVideoQuickstart/ViewController.m]: I'm not sure that it needs to be changed. The tracks are collectively the local media which we are sharing.

**Pull request merged**